src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/TriviaEngine/AbstractTriviaFormatter.cs

```diff
@@ -279,27 +289,41 @@ private LineColumn FormatTrivia<T>(Formatter<T> formatter, WhitespaceAppender<T>
             var implicitLineBreak = false;
 
             var list = new TriviaList(this.Token1.TrailingTrivia, this.Token2.LeadingTrivia);
+            var i = -1;
+            var previousLineColumn = new LineColumn();
             foreach (var trivia in list)
             {
+                i += 1;
                 if (trivia.RawKind == 0)
                 {
                     continue;
                 }
 
                 if (IsWhitespaceOrEndOfLine(trivia))
                 {
+
+                    existingWhitespaceDelta = existingWhitespaceDelta.With(
+                       GetLineColumnOfWhitespace(
+                           lineColumn,
+                           previousTrivia,
+                           previousWhitespaceTrivia,
+                           existingWhitespaceDelta,
+                           trivia));
```

Comment:
can you add a comment in the code waht this is for?

Reply:
It was in original code I just move it up 4 lines before an if.